shallow_backup/reinstall.py

```diff
@@ -64,6 +63,13 @@ def reinstall_packages_sb(packages_path):
 	"""
 	Reinstall all packages from the files in backup/installs.
 	"""
+	if not os.path.isdir(packages_path):
+		print_red_bold('No package backups found.')
```

Comment:
I think this should exit after no package backups are found. If you want to backup, you should have to go through the main menu, and I want to keep backup and reinstallation clearly separate.

Reply:
If this change is removed, also remember to remove the unused imports at the top.

Reply:
Good fix. In taking another look at the code, we should add another condition to check for the packages backup folder existing, but having nothing inside of it.

Reply:
3783008ac84fb8d1ec1d813b7ee9c2e50e0a8cc3